frontend/ui/elements/screen_rotation_menu_table.lua

```diff
@@ -24,9 +26,30 @@ return {
             })
         end
 
+        if Device:hasGSensor() then
+            table.insert(rotation_table, {
+                text = _("Lock auto rotation to current orientation"),
+                help_text = _([[
+When checked, the gyro will only be honored when switching between the two inverse variants of your current rotation,
+i.e., Portrait <-> Inverted Portrait OR Landscape <-> Inverted Landscape.
+Switching between (Inverted) Portrait and (Inverted) Landscape will be inhibited.
+If you need to do so, you'll have to use the UI toggles.]]),
+                enabled_func = function()
+                    return G_reader_settings:nilOrFalse("input_ignore_gsensor")
+                end,
+                checked_func = function()
+                    return G_reader_settings:isTrue("input_lock_gsensor")
+                end,
+                callback = function()
+                    G_reader_settings:flipNilOrFalse("input_lock_gsensor")
+                    Device:lockGSensor(G_reader_settings:isTrue("input_lock_gsensor"))
+                end,
+            })
+        end
+
         table.insert(rotation_table, {
             text = _("Keep file browser rotation"),
-            help_text = _("When checked the rotation of the file browser and the reader will not affect each other"),
+            help_text = _("When unchecked, the rotation of the file browser and the reader will not affect each other"),
```

Comment:
It is correct the way it is, it should say checked

Reply:
Ah, that's indeed a problem, because that menu label is now nonsensical ^^.